admin/src/components/ContributionMessages/slots/ContributionMessagesListItem.spec.js

```diff
@@ -9,50 +9,86 @@ describe('ContributionMessagesListItem', () => {
   const mocks = {
     $t: jest.fn((t) => t),
     $d: jest.fn((d) => d),
-    $store: {
-      state: {
-        moderator: {
-          id: 107,
-        },
-      },
-    },
   }
 
-  const propsData = {
-    contributionId: 42,
-    state: 'PENDING',
-    message: {
-      id: 111,
-      message: 'asd asda sda sda',
-      createdAt: '2022-08-29T12:23:27.000Z',
-      updatedAt: null,
-      type: 'DIALOG',
-      userFirstName: 'Peter',
-      userLastName: 'Lustig',
-      userId: 107,
-      __typename: 'ContributionMessage',
-    },
-  }
+  describe('if message author has moderator role', () => {
+    const propsData = {
+      contributionId: 42,
+      state: 'PENDING',
+      message: {
+        id: 111,
+        message: 'Lorem ipsum?',
+        createdAt: '2022-08-29T12:23:27.000Z',
+        updatedAt: null,
+        type: 'DIALOG',
+        userFirstName: 'Peter',
+        userLastName: 'Lustig',
+        userId: 107,
+        isModerator: true,
+        __typename: 'ContributionMessage',
+      },
+    }
 
-  const Wrapper = () => {
-    return mount(ContributionMessagesListItem, {
-      localVue,
-      mocks,
-      propsData,
-    })
-  }
+    const ModeratorItemWrapper = () => {
+      return mount(ContributionMessagesListItem, {
+        localVue,
+        mocks,
+        propsData,
+      })
+    }
 
-  describe('mount', () => {
-    beforeEach(() => {
-      wrapper = Wrapper()
-    })
+    describe('mount', () => {
+      beforeAll(() => {
+        wrapper = ModeratorItemWrapper()
+      })
+
+      it('has a DIV .text-right.is-moderator', () => {
+        expect(wrapper.find('div.text-right.is-moderator').exists()).toBe(true)
+      })
 
-    it('has a DIV .contribution-messages-list-item', () => {
-      expect(wrapper.find('div.contribution-messages-list-item').exists()).toBe(true)
+      it('props.message.default', () => {
```

Comment:
```suggestion
      it('has empty object as props.message.default', () => {
```

Maybe?
But should it have a default, because it's required?
See below 👇🏼